### ISSUE TITLE
Question - recently-updated-shows behavior

### ISSUE THREAD

**Issue by jonathanantoine** [trakt/api-help#150]
Hello,

I am wondering when a show is included in this endpoint ? https://trakt.docs.apiary.io/#reference/shows/watched/get-recently-updated-shows

When any of it's info change ? Also when any of it's episode info is changed ?

Regards

**Comment by tysonkerridge** [comment]
This might be what you’re after: https://github.com/trakt/api-help/issues/37#issuecomment-472105218

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jonathanantoine** [comment]
This is it, thanks! This should definitively be in the documentation !

I tried with today dates and I got 20 pages of results... Is it a normal behavior @rectifyer ? It looks like a lot of request to get the full list of updated shows (I am trying to save the ApI some calls 😁).

**Comment by jonathanantoine** [comment]
up :)

**Comment by jonathanantoine** [comment]
Do you have any idea of the answer @tysonkerridge  who seems to know a lot of things related to the Api too 😁?

**Comment by tysonkerridge** [comment]
Hmm, nah I’m not too sure about what’s expected there. But I imagine it’s normal if you’re using a rather small page size? That end point is returning info about all possible shows in Trakt so it’s not surprising if there’s a lot, especially when the data from TVDB and TMBD is likely changing a lot each day too. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by jonathanantoine** [comment]
It was using the biggest available page size. If I have 100 shows to sync, it would be an Optimization (less load on the API) only if I have less call to make to the changes endpoint I guess.

**Comment by rectifyer** [comment]
@jonathanantoine yes, there is usually going to be a high number of updates each day since it includes anything changed on TVDB and TMDB in the last 24 hours. What is your suggestion about optimizing the `updates` endpoint?

**Comment by jonathanantoine** [comment]
@rectifyer I have 3 ideas : 
- An endpoint providing the show changed for a provided users.
- An endpoint where I would be able to provide you a list of show id and you would returns me only the ids of the changed one.
- Let us use biggest pagination page size : 1000 or 5000 ?

In all case I would love to send UTC date **and time** and not only date as the current behavior.
